profiles/cli.py

```diff
@@ -18,3 +31,28 @@ def __init__(self, *repositories: CanBeCleared) -> None:
     def run(self) -> None:
         for each in self.repositories:
             each.clear()
+
+
+class SetOrcidWebhooksCommand(Command):
+    NAME = 'set-orcid-webhooks'
+
+    def __init__(self, profiles: Profiles, orcid: Dict[str, str], orcid_client: OrcidClient,
+                 uri_signer: URLSafeSerializer) -> None:
+        super(SetOrcidWebhooksCommand, self).__init__()
+        self.profiles = profiles
+        self.orcid = orcid
+        self.orcid_client = orcid_client
+        self.uri_signer = uri_signer
+
+    # weird decoration made by the superclass that wraps this method
+    # not going to lose sleep over this for now,
+    # let's see whether we get more commands
+    # and we keep flask_script in the long term
+    # pylint: disable=method-hidden
+    def run(self) -> None:
+        access_token = self.orcid.get('webhook_access_token')
+
+        for profile in self.profiles.list():
+            uri = url_for('webhook._update', payload=self.uri_signer.dumps(profile.orcid),
+                          _external=True)
+            self.orcid_client.set_webhook(profile.orcid, uri, access_token)
```

Comment:
it would be useful to log/print something to understand how many profiles have been involved, how quickly it is progressing

Reply:
`OrcidClient.set_webhook()` already prints what its doing, but there's no count yet. Will add one.